### PR TITLE
Issue 681 - Upgrade plotly.js to 1.50.1

### PR DESCRIPTION
**Pull request by Marc-Andre-Rivet**

### CODE REVIEW

**Comment by alexcjohnson** [comment]
Want to add a test for https://github.com/plotly/dash-core-components/issues/672?

**Comment by Marc-Andre-Rivet** [comment]
Digging through this, the issue / full fix is not solely in Plotlyjs.
https://github.com/plotly/dash-core-components/issues/672#issuecomment-542784762

File: tests/integration/graph/test_graph_basics.py [hunk 1]
```diff
@@ -88,3 +88,48 @@ def selected_df_figure(selection):
...
+                    'shape': 'spline'
+                }
+            }],
+        }
+
+    dash_dcc.start_server(app)
+
+    dash_dcc.wait_for_element('#my-graph .main-svg')
+
+    assert not dash_dcc.get_logs()
```
Comment (Marc-Andre-Rivet) [inline tests/integration/graph/test_graph_basics.py]: Make sure that when rendering a graph as a callback-child of a loading component, no error occurs

**Comment by Marc-Andre-Rivet** [comment]
Failed run without the dcc fix: https://circleci.com/gh/plotly/dash-core-components/10410
Successful run with the dcc fix: https://circleci.com/gh/plotly/dash-core-components/10421

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,14 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [Unreleased]
+### Updated
+- Upgraded plotly.js to 1.50.1 [#681](https://github.com/plotly/dash-core-components/issues/681)
+  - Path release [1.50.1](https://github.com/plotly/plotly.js/releases/tag/v1.50.1) containing several bug fixes.
```
Comment (alexcjohnson) [inline CHANGELOG.md]:
```suggestion
  - Patch release [1.50.1](https://github.com/plotly/plotly.js/releases/tag/v1.50.1) containing several bug fixes.
```

**Pull request merged**